src/proxy-ui-api/frontend/src/views/Clients/ServiceClients/ServiceClientAccessRights.vue

```diff
@@ -110,24 +122,47 @@ export default Vue.extend({
         .catch( (error: any) =>
           this.$store.dispatch('showError', error));
 
+      this.fetchAccessRights();
+
+    },
+    fetchAccessRights(): void {
       api
         .get(`/clients/${this.id}/service-clients/${this.serviceClientId}/access-rights`)
         .then( (response: any) => this.accessRights = response.data)
         .catch( (error: any) =>
           this.$store.dispatch('showError', error));
-
     },
     close() {
       this.$router.go(-1);
     },
-    remove() {
-      // NOOP
+    remove(accessRight: AccessRight) {
```

Comment:
Add missing return types to methods. 
In other respects frontend is ok.

Reply:
Fixed